src/Context.ts

```diff
@@ -3,6 +3,7 @@ import { fromRpcHttp, fromTypedData, getNormalizedBindingData, getBindingDefinit
 import { AzureFunctionsRpcMessages as rpc } from '../azure-functions-language-worker-protobuf/src/rpc';

```

Comment:
nit: you can import in the above `from './converters'`, because converters/index.js exports everything in `RpcConverters`